src/main/resources/custom-events.sk

```diff
@@ -0,0 +1,38 @@
+
+# This file allows you to register custom events that aren't already registered under Skript.
+# The event classes you want to register must exist in the server class loader. So events from Paper/Spigot.
+# If you want to register an event from a plugin API, you must edit the plugin.yml in the jar of Skript to include:
+#
+# softdepend: [YourPlugin, SQLibrary, Vault, WorldGuard, Residence, PreciousStones, GriefPrevention]
+#
+# Reminder to maintain any existing defined plugins in this array as this example should not be directly copied.
+#! Additions or changes to this file requires a server restart.
+
+#! Internal version of this configuration. Don't modify.
+version: 1
+
+# The format must include a section where the section name is the event name.
+# The event section must include:
+# 	description - describing the event
+#	class - the class name to search for
+# 	pattern - the syntax pattern of the event
+# The optional pattern [on] gets prefixed on every Skript event.
+# Pattern cannot have expressions inside currently.
+# event-values of subclass events will be automatically applied.
+# If you need more info, make an issue report at https://github.com/SkriptLang/Skript/issues
+
+# Examples are placed below. The 'enabled' node is not required, it's just to disable examples.
+events:
+	# [on] beacon effect apply
+	beacon effect:
+		enabled: false
+		description: "Called when entities receive potion effects from a beacon"
+		class: com.destroystokyo.paper.event.block.BeaconEffectEvent
+		pattern: beacon effect apply [to player[s]]
+
+	# [on] campfire lit
```

Comment:
This seems like a really confusing and unhelpful comment, since the _entry_ name is `campfire lit` but the pattern below is actually `campfire start[ing] cooking` which is what is registered by the CustomEvents class, so it wouldn't make an `[on] campfire lit` pattern.